From 220de240b48f26d77502f041fe45a97c9cc94fad Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Sat, 28 Jul 2018 06:30:20 +0800
Subject: [PATCH 4552/5242] MLK-19080 ARM64: dts: freescale: imx8x: adjust
 passive trip point setting

commit  aba36838120118ccd9541752b162ec8b9f83322f from
https://source.codeaurora.org/external/imx/linux-imx.git

Adjust passive trip point temperature to be 20 degree C
below than the critical trip point temperature on i.MX8X
platforms.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Reviewed-by: Ye Li <ye.li@nxp.com>
(cherry picked from commit cefa63c1b9873e5e60f4db1e77bfecfaf18ff799)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi      |    4 ++--
 .../boot/dts/freescale/fsl-imx8qm-device.dtsi      |   10 +++++-----
 arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts   |    2 +-
 arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts  |    2 +-
 4 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
index dae0b08..0473462 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8dx.dtsi
@@ -1377,7 +1377,7 @@
 			thermal-sensors = <&tsens 0>;
 			trips {
 				cpu_alert0: trip0 {
-					temperature = <85000>;
+					temperature = <107000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
@@ -1403,7 +1403,7 @@
 			status = "disabled";
 			trips {
 				drc_alert0: trip0 {
-					temperature = <85000>;
+					temperature = <107000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
index 6cb4313..00177b8 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-device.dtsi
@@ -1429,7 +1429,7 @@
 			thermal-sensors = <&tsens 0>;
 			trips {
 				cpu_alert0: trip0 {
-					temperature = <85000>;
+					temperature = <107000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
@@ -1454,7 +1454,7 @@
 			thermal-sensors = <&tsens 1>;
 			trips {
 				cpu_alert1: trip0 {
-					temperature = <85000>;
+					temperature = <107000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
@@ -1479,7 +1479,7 @@
 			thermal-sensors = <&tsens 2>;
 			trips {
 				gpu_alert0: trip0 {
-					temperature = <85000>;
+					temperature = <107000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
@@ -1497,7 +1497,7 @@
 			thermal-sensors = <&tsens 3>;
 			trips {
 				gpu_alert1: trip0 {
-					temperature = <85000>;
+					temperature = <107000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
@@ -1515,7 +1515,7 @@
 			thermal-sensors = <&tsens 4>;
 			trips {
 				drc_alert0: trip0 {
-					temperature = <85000>;
+					temperature = <107000>;
 					hysteresis = <2000>;
 					type = "passive";
 				};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
index 8902bf3..33e660f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-mek.dts
@@ -1420,7 +1420,7 @@
 		thermal-sensors = <&tsens 5>;
 		trips {
 			pmic_alert0: trip0 {
-				temperature = <80000>;
+				temperature = <110000>;
 				hysteresis = <2000>;
 				type = "passive";
 			};
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
index f93d90b..6138b6f 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qxp-mek.dts
@@ -1234,7 +1234,7 @@
 		thermal-sensors = <&tsens 2>;
 		trips {
 			pmic_alert0: trip0 {
-				temperature = <80000>;
+				temperature = <110000>;
 				hysteresis = <2000>;
 				type = "passive";
 			};
-- 
1.7.9.5

